src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/_utils.py

```diff
@@ -91,6 +91,22 @@ def get_registry_by_name(cli_ctx, registry_name, resource_group_name=None):
     return client.get(resource_group_name, registry_name), resource_group_name
 
 
+def get_registry_by_login_server(cli_ctx, login_server):
+    """Returns a Registry object for the specified login server.
+    :param str login_server: The login server of the container registry.
+    """
+    login_server_lower = login_server.lower()
+
+    client = get_acr_service_client(cli_ctx).registries
+    registry_list = client.list()
+
+    elements = [item for item in registry_list if item.login_server.lower() == login_server_lower]
+
+    if len(elements) != 1:
+        return None
```

Comment:
Why is this?  Are there ever more than 1?  If so, why return nothing in that case?

Reply:
Good catch. Fixed it.